presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/jmx/StatisticsAwareJdbcClient.java

```diff
@@ -56,12 +55,6 @@ public StatisticsAwareJdbcClient(JdbcClient delegate)
         this.delegate = requireNonNull(delegate, "delegate is null");
     }
 
-    @Override
-    protected JdbcClient delegate()
```

Comment:
if you keep delegate (as private), the diff will be small & nice

Reply:
I thought about that, but keeping a useless method just to make the diff nice seems lame. (and you know how much I like nice diffs)

Reply:
I updated it to keep the method